packaging/linux/deb/template_rules.go

```diff
@@ -99,7 +108,8 @@ func (w *rulesWrapper) OverridePerms() fmt.Stringer {
 		checkPerms(artifacts.Libexec) ||
 		checkPerms(artifacts.DataDirs) ||
 		checkDirPerms(artifacts.Directories.GetConfig()) ||
-		checkDirPerms(artifacts.Directories.GetState())
+		checkDirPerms(artifacts.Directories.GetState()) ||
+		checkSymlinkOwnership(artifacts.Links)
```

Comment:
Looks like this (and the function being called) is leftover from a previous iteration.

```suggestion
		checkDirPerms(artifacts.Directories.GetState())
```